include/generic_sort.h

```diff
@@ -187,4 +198,20 @@ class UninterpretedGenericSort : public GenericSort
   SortVec param_sorts;  
 };
 
+class GenericDatatypeSort : public GenericSort
+{
+ public:
+  GenericDatatypeSort(const Datatype & dt);
+  ~GenericDatatypeSort();
+  // Returns the name of the datatype
+  std::string get_sort_name();
+  Datatype get_datatype() const override;
+  bool compare(const Sort & s) const override;
+  std::string compute_string() const;
```

Comment:
How come thid doesn't `override`?

Reply:
sort.h does not declare a compute_string function, and so generic_sort doesn't override anything for its compute_string() function. And so genericDatatypeSort doesn't have a 'base' virtual function to just override.

Reply:
I suggest we change that. Maybe add `= 0` in `GenericSort` for that funciton, to make sure that ehre is a default empty unction, and then mark it here is `override`.